app/scripts/directives/rule.js

```diff
@@ -129,15 +132,13 @@ angular.module('confRegistrationWebApp').directive('rule', function () {
           case 'genderQuestion':
             return ['M', 'F'];
           case 'yearInSchoolQuestion':
-            return [
-              'Freshman',
-              'Sophomore',
-              'Junior',
-              'Senior',
-              'Graduate Student',
-            ];
+            return block.content.staticChoices;
```

Comment:
Currently, the Graduation Date is showing no Question rules, is that intentional?

Reply:
Graduation date should have question rules. The question has to be below the graduation date question. It is working for me.

Reply:
Questions related to dates aren't in this switch. This is only for questions that have choices to choose from.